WebApps/CNS.Auth.Web/Services/CNSCertificateService.cs

```diff
@@ -80,6 +89,12 @@ public Task<ClaimsPrincipal> GetClaimsPrincipal(X509Certificate2 cert, Cancellat
 			);
 		}
 
+		/// <summary>
+		/// Validate certificate with thumbprint for CNS Certificate policies
+		/// </summary>
+		/// <param name="cert">X509 Certificate</param>
+		/// <param name="cancellationToken">Cancellation Token</param>
+		/// <returns>Bool value for validation</returns>
```

Comment:
```suggestion
		/// <summary>
		/// Validates the provided X509Certificate2 checking if it has the required CNS certificate policies
		/// </summary>
		/// <param name="cert">The X509Certificate2 to check</param>
		/// <param name="cancellationToken">Cancellation Token</param>
		/// <returns>true if the X509Certificate2 has the required CNS certificate policies. false otherwise</returns>
```